src/validation-errors/enum.js

```diff
@@ -2,17 +2,51 @@ import chalk from 'chalk';
 import leven from 'leven';
 import pointer from 'jsonpointer';
 import printJson from '../json/print';
-
+import findErrorPosition from '../json/find-error-position';
 import BaseValidationError from './base';
 
 export default class EnumValidationError extends BaseValidationError {
   print(schema, data) {
     const output = [];
     const { message, dataPath, params: { allowedValues } } = this.options;
+    const bestMatch = this.findBestMatch(data);
+
     output.push(chalk`{red {bold ENUM} ${message}}`);
     output.push(chalk`{red (${allowedValues.join(', ')})}\n`);
 
+    return output.concat(
+      printJson(data, dataPath, { indent: this.indent })(() => {
+        if (bestMatch) {
```

Comment:
If `bestMatch` value is `0` or `false` then this will fail.

Reply:
fixed dc6d89a